README.md

```diff
@@ -0,0 +1,79 @@
+# Environment Variables Injector
+
+This extension is designed to help users export snap options configured with the [snap set command](https://snapcraft.io/docs/configuration-in-snaps) into environment variables.
+
+Additionally, this extension supports environment files, which we refer to as `envfile` for brevity.
```

Comment:
The introduction needs improvements, keeping the following points in mind:
- The extension is to inject environment variables into snaps
- The extension offers two ways to do it
  - Via snap configuration options
  - Via environment files or env files

Reply:
Addressed on https://github.com/locnnil/env-inject-doc/pull/1/commits/1f0181a11f9680f197444887ec495a6854d5e13b! Thanks!